pressbooks/pressbooks-multi-institution
# fix: re order institutions menu item

This PR addresses https://github.com/pressbooks/pressbooks/issues/3638.

This is not a complete solution to our sidebar menu problem, and it does not address a flexible approach. 
This PR:
- Adds the `Institutions` item at the end of the sidebar menu (without specifying any order in particular).
- Reorders the menu to ensure that the new `Institutions` menu item is placed in the 4 position of whatever menu is defined.
- Removes the koko analytics sub menu item under Stats for Institutional Managers, since they don't have access to it.
- Renames `Network Stats` submenu item to `<Institution Name> Stats`.

It requires https://github.com/pressbooks/pressbooks/issues/3638

### Testing case
- Make sure you have https://github.com/pressbooks/pressbooks/issues/3638 in place.
- Log in as a super admin.
- Go to any network admin page like Dashboard, or Network users page.
- Institutions menu item should be in the 4th place (under Users). The menu item Page, should be under Appearance as before.
- Go to a Rootsite page like Page or Root Site Users.
- Institutions menu item should be in the 4th place (under Users). The menu item Page, should be under Appearance as before, just like the Network Admin pages.
